src/app/app.component.html

```diff
@@ -122,13 +122,14 @@ <h2>Step {{ firstStep.stepNumber }}: {{ firstStep.stepName }}</h2>
                                                 aria-expanded="false"
                                             >
                                                 Select a
-                                                {{ firstBackground.principleCategory }} principle
+                                                {{ selectedBackground.principleCategory }} principle
                                             </button>
                                             <ul class="dropdown-menu" aria-labelledby="backgroundPrincipleDropdown">
                                                 <li>
                                                     <button
                                                         class="dropdown-item btn btn-outline"
-                                                        *ngFor="let principle of getPrinciplesFromCategory(firstBackground.principleCategory)"
+                                                        *ngFor="let principle of getPrinciplesFromCategory(selectedBackground.principleCategory)"
```

Comment:
usually we want to stay away from function calls in the template, cause they tend to be called a lot more than once due to the way Angular change detection works.

next we meet, we are going to debug this to see how it is performing, and it will be a good opportunity to work on debugging skills